_posts/2021-01-08-2020-review.md

```diff
@@ -0,0 +1,254 @@
+better, so we worked on that:
+
+* Improved pushdown of complex operations into connectors, including
+  [aggregation and topN pushdown](/docs/current/optimizer/pushdown.html). We’ve
+  extended several connectors with these new capabilities.
+* [Dynamic filtering and partition pruning]({% post_url
+  2020-06-14-dynamic-partition-pruning %}), which can improve performance of
+  highly selective joins manyfold.
+* Cost-based decisions for queries containing `IN <subquery>` in `WHERE` clause
+* Information_schema performance improvements, which benefit third-party BI
+  tools that need to inspect table metadata (e.g., DBeaver, Datagrip, etc).
+* [Faster queries on nested data in Parquet and ORC]({% post_url
+  2020-08-14-dereference-pushdown %})
+* Faster and more accurate `approx_percentile`, based on t-digest data structure
+* Support of Bloom filters in ORC
+* Optimized Parquet writer
+
+## Security
+
+The more data you access with Trino, the more it becomes critical to secure it.
+With that in mind we added a lot of improvements:
+
+* The [Web UI](/docs/current/admin/web-interface.html) now requires
+  authentication. Various actions such as viewing query details, killing
+  queries, etc., are protected with authorization checks based on the identity
+  of the user. Additionally, the UI now supports OAuth2 for user identification.
+* External and internal APIs are now properly secured with authentication and
+  authorization checks. Importantly, this fixes a [CVE reported
+  vulnerability](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-15087)
+  that affects in all older and other versions of Presto®]
+*  A [new mechanism to externalize secrets in configuration
+   files](/docs/current/security/secrets.html) that makes it easier to integrate
+   with third-party secret managers and deployment tools.
+* Support for JSON Web Key (JWK) authentication and [pluggable certificate
+  authenticators](/docs/current/develop/certificate-authenticator.html)
+* [Salesforce authenticator](docs/current/security/salesforce.html)
+* The query engine and access control SPIs now support injecting row filters and
+  column masks.
+* New syntax for managing permissions (`GRANT/REVOKE` on schema,
+  `ALTER TABLE/SCHEMA/VIEW ... SET AUTHORIZATION`)
+
+## Supported data source
+
+Trino empowers you to use one platform to access all data sources. Connectors
+enable this and we added numerous new connectors:
+
+* [Iceberg](/docs/current/connector/iceberg.html)
+* [Prometheus](/docs/current/connector/prometheus.html)
+* [Oracle](/docs/current/connector/oracle.html)
+* [Pinot](/docs/current/connector/pinot.html)
+* [Druid](/docs/current/connector/druid.html)
+* [BigQuery](/docs/current/connector/bigquery.html)
+* [MemSQL](/docs/current/connector/memsql.html)
+
+All other connectors received a large host of improvements. Let’s just look at
+two popular connectors:
+
+### Hive connector for HDFS, S3 and cloud object storage system
```

Comment:
spell out  GCS and Azure storage 

while Azure support is old, over last year we confirmed compatibility by adding & enabling continues integration with azure storages (i think all three)

Reply:
its in the bullet list ... title would get a bit long .. not sure